examples/data-objects/table-document/README.md

```diff
@@ -5,6 +5,10 @@
 <!-- prettier-ignore-start -->
 <!-- NOTE: This section is automatically generated using @fluid-tools/markdown-magic. Do not update these generated contents directly. -->
 
+**NOTE: This package is an example leveraging the [Fluid Framework](https://fluidframework.com).**
```

Comment:
We know for legacy reasons this one is actually part of our API surface, but I think keeping the notice is still fine?

Reply:
Yeah. My thinking was that this would give a bit of extra warning to folks who look at it. We may *actually* be offering support for this, but we shouldn't advertise that fact IMO.